dist/origin-web-catalogs.css

```diff
@@ -60,37 +60,56 @@
   padding-bottom: 40px;
 }
 .landing-main-area {
-  margin: 15px 320px 15px 20px;
+  margin: 15px 20px;
   padding-right: 10px;
 }
+@media (min-width: 768px) {
```

Comment:
We should be using the Bootstrap variables for media queries.

http://getbootstrap.com/css/#grid-media-queries

```css
@media (min-width: @screen-sm-min) { ... }
```